tools/provision/chocolatey/osquery_utils.ps1

```diff
@@ -225,3 +225,39 @@ function Start-OsqueryProcess {
     }
   }
 }
+
+# A helper function to derive the latest VS install
+function Get-VSInfo {
+
+  # Attempt to make use of vswhere to derive the build tools scripts
+  $vswhere = (Get-Command 'vswhere').Source
+  $vswhereArgs = @('-latest', '-legacy')
+  $vswhereOut = (Start-OsqueryProcess $vswhere $vswhereArgs).stdout
+  $vsinfo = "" | Select-Object -Property location,version
```

Comment:
This is a pretty sweet hack! However let's be more explicit in our object creation:
```
$vsinfo = New-Object -TypeName psobject
$vsinfo | Add-Member -MemberType NoteProperty -Name version -Value ''
$vsinfo | Add-Member -MemberType NoteProperty -Name location -Value ''
```